Dockerfile

```diff
@@ -1,15 +1,21 @@
-FROM python:3
+FROM python:alpine
 
 WORKDIR /usr/src/app
 
+RUN apk add gcc musl-dev libffi-dev openssl-dev libxml2-dev libxslt-dev file llvm-dev make g++
+
 COPY requirements.txt ./
+
+RUN pip install wheel cryptography gunicorn pymysql
```

Comment:
``--no-cache-dir``

Why are those deps not in the requirements.txt?

Reply:
Those dependencies can be found in [here](https://github.com/pluja/Yotter/blob/dev-indep/SELF-HOSTING.md#manual-installation) looks like they're only used for production environments.